config/routes.rb

```diff
@@ -1,60 +1,24 @@
-Rails.application.routes.draw do
-
-  resources :companies do
-    resources :jobs
-  end
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
-
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+parent_needed = [:new, :create, :update]
+actions = [:create, :update, :destroy]
```

Comment:
Think we talked a little bit about this: would likely prefer to see these in each of the routes rather than stored as variables here. However, really like that you're able to see that this is Ruby and you can use the things you learned in M1.